techlibs/xilinx/synth_xilinx.cc

```diff
@@ -213,9 +209,8 @@ struct SynthXilinxPass : public ScriptPass
 				continue;
 			}
 			if (args[argidx] == "-iopad") {
-				iopad = true;
 				continue;
-			}
+			}			
```

Comment:
Extra spacing here

Reply:
addressed in 436fea9e6990c66369d7c30b571920ae115efb44